### PR TITLE
Feat pages - added wishlist,cart,auth

### PR DESCRIPTION
**Pull request by vivekgudipudi**
image
image
image
image
image

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Though the PR was very big but the code is neatly structured and good to merge
Appreciation: The UI looks amazing

File: src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const navigate = useNavigate();
+
+    const { setSearchText } = useProductContext();
+    const loginClickHandler = () => { if(isLoggedIn){
+        window.location.reload(false);
+        setIsLoggedIn(!isLoggedIn);
+        
+    }
+    // else{
+    //     // localStorage.clear()
```
Comment (srishtigautam21) [inline src/components/NavBar.js:23]: Remove commented code

Comment (vivekgudipudi) [reply src/components/NavBar.js:23]: sure, Thanks for the review

**Review by shashikantmaurya0007 — APPROVED** [review]
Srishti has suggested few changes .apart from that everything looks good

**Pull request merged**